.github/workflows/ci.yml

```
@@ -37,6 +37,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
+    
+    # Chrome Stable is already installed by default.
+    - name: Install Chrome ToT
+      run: bash lighthouse-core/scripts/download-chrome.sh
+      env:
```

```suggestion
      working-directory: /home/runner
      env:
```

optional, feels weird to download this in repo root, might as well just download it where we want it